extensions/components/com_redcore/site/layouts/translation/fields/state.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
all those layout files should be reused (if they are similar to their admin counterparts) so maybe load admin layout files from here